flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/iomanager/BufferFileWriterReaderTest.java

```diff
@@ -199,25 +198,26 @@ private int getNextMultipleOf(int number, int multiple) {
 	}
 
 	private Buffer createBuffer() {
-		return new Buffer(MemorySegmentFactory.allocateUnpooledSegment(BUFFER_SIZE), BUFFER_RECYCLER);
+		return new NetworkBuffer(MemorySegmentFactory.allocateUnpooledSegment(BUFFER_SIZE), BUFFER_RECYCLER);
 	}
 
-	public static int fillBufferWithAscendingNumbers(Buffer buffer, int currentNumber) {
-		MemorySegment segment = buffer.getMemorySegment();
+	static int fillBufferWithAscendingNumbers(Buffer buffer, int currentNumber, int size) {
+		assert size % 4 == 0;
```

Comment:
`checkState`?

Reply:
`checkArgument` - but why not (although iirc, assertions should be enabled in tests)